lib/Getopt/Std.pm

```diff
@@ -251,6 +257,7 @@ sub getopts ($;$) {
 		    $$hash{$first} = $rest;
 		}
 		else {
+            no strict 'refs';
```

Comment:
indentation looks weird, not sure if this is the original or your patch, but they are not matching (same with the next line)